vunit/sim_if/ghdl.py

```diff
@@ -273,35 +273,39 @@ def compile_vhdl_file_command(self, source_file):
         return cmd
 
     def _get_command(  # pylint: disable=too-many-branches
-        self, config, output_path, elaborate_only, ghdl_e, wave_file
+        self, config, output_path, elaborate_only, ghdl_e, wave_file, precompiled=None
     ):
         """
         Return GHDL simulation command
         """
         cmd = [str(Path(self._prefix) / self.executable)]
-
-        if ghdl_e:
-            cmd += ["-e"]
+        if precompiled:
+            self._vhdl_standard = config._design_unit.source_file._vhdl_standard
```

Comment:
Why does the standard need to be set here?

Reply:
The has_valid_exit_code() function accesses the self._vhdl_standard, where it crashes because the 'other' parameter in the VHDLStandard object for __lt__(self, other) is None.

I guess it could be set elsewhere? But since I'm not well-versed with this code base other than the user API, do you have a suggestion as to where?